cmake/multi_image.cmake

```diff
@@ -266,17 +266,23 @@ function(add_child_image_from_source)
 
   if (ACI_DOMAIN)
     add_custom_target(${ACI_NAME}_flash
-                      COMMAND
-                      ${CMAKE_COMMAND} --build ${CMAKE_BINARY_DIR}/${ACI_NAME}
-                      --target flash
-                      DEPENDS
-                      ${ACI_NAME}_subimage
-    )
+      COMMAND
+      ${CMAKE_COMMAND} --build ${CMAKE_BINARY_DIR}/${ACI_NAME}
+      --target flash
+      DEPENDS
+      ${ACI_NAME}_subimage
+      )
```

Comment:
I know we still don't have official CMake coding style, but I liked the previous `)` alignment that followed `add_custom_command(`.

Reply:
But the rest of the file follows this style, do you still think I should revert the changes?